### PR TITLE
Use Travis CI

### PR DESCRIPTION
**Pull request by nerg4l**
Add Travis CI configuration to the project to run tests on different Laravel/PHP versions automatically.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  directories:
+    - $HOME/.composer/cache
+
+php:
+  - 7.1
+  - 7.2
+  - 7.3
+
+env:
+  - ILLUMINATE_VERSION=5.5.* TESTBENCH_VERSION=3.5.* PHPUNIT_VERSION=^6.0
```
Comment (axlon) [inline .travis.yml]: Currently these lines do nothing, you’re setting environment variabels but you’re not using them. I’d take the geo-routes Travis file as an example, you will see some composer require statements to install to correct versions

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - ILLUMINATE_VERSION=5.6.* TESTBENCH_VERSION=3.6.* PHPUNIT_VERSION=^7.0
+  - ILLUMINATE_VERSION=5.7.* TESTBENCH_VERSION=3.7.* PHPUNIT_VERSION=^7.0
+
+before_install:
+  - travis_retry composer self-update
+  - travis_retry composer update --no-interaction --prefer-source
```
Comment (axlon) [inline .travis.yml]: This should be in your install step, before install is not meant for installing your dependencies. Also why are you passing —prefer-source, wouldn’t someone using this library be using dist versions of our dependencies. You want to replicate as close to a real world scenario as possible

File: .travis.yml [hunk 3]
```diff
@@ -14,9 +15,20 @@ env:
...
+  exclude:
+    - php: 7.0
+      env: ILLUMINATE_VERSION=5.6.* TESTBENCH_VERSION=3.6.* PHPUNIT_VERSION=^7.0
+    - php: 7.0
+      env: ILLUMINATE_VERSION=5.7.* TESTBENCH_VERSION=3.7.* PHPUNIT_VERSION=^7.0
+
 before_install:
-  - travis_retry composer self-update
-  - travis_retry composer update --no-interaction --prefer-source
+  - composer require illuminate/support:${ILLUMINATE_VERSION} illuminate/validation:${ILLUMINATE_VERSION} --no-update
```
Comment (yak0d3) [inline .travis.yml]: There is no need to require the `illuminate/validation` dependency instead the `illuminate/http` should be included

**Comment by nerg4l** [comment]
I did everything you requested

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+php:
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
+
+env:
+  - ILLUMINATE_VERSION=5.5.* TESTBENCH_VERSION=3.5.* PHPUNIT_VERSION=^6.0
+  - ILLUMINATE_VERSION=5.6.* TESTBENCH_VERSION=3.6.* PHPUNIT_VERSION=^7.0
+  - ILLUMINATE_VERSION=5.7.* TESTBENCH_VERSION=3.7.* PHPUNIT_VERSION=^7.0
```
Comment (axlon) [inline .travis.yml]: No Laravel 5.8?

**Pull request merged**